WrightTools/artists/_quick.py

```diff
@@ -309,7 +309,23 @@ def quick2D(
             ax.contourf(d, channel=channel_index, cmap=cmap, levels=levels)
         # contour lines ---------------------------------------------------------------------------
         if contours:
-            raise NotImplementedError
+            # get contour levels
+            # force top and bottom contour to be just outside of data range then add two contours
+            if channel.signed:
+                if contours_local:
+                    limit = channel.mag()
+                else:
+                    limit = data_channel.mag()
+                contour_levels = np.linspace(
+                    -limit + channel.null - 1e-10, limit + channel.null + 1e-10, contours + 2
+                )
+            else:
+                if contours_local:
+                    limit = np.nanmax(zi)
+                else:
+                    limit = data_channel.mag()
+                contour_levels = np.linspace(channel.null - 1e-10, limit + 1e-10, contours + 2)
```

Comment:
Why are we avoiding a contour at channel.null?

I get avoiding the maximum and the min/max in signed, but why null?

Reply:
Were we not to do this, if there is data at (or below) null, then contour would be present and unhelpful.

Reply:
TBH, this is how it was written in `mpl_2D` and I just kept the idea.

Reply:
if there's data below null here, it will be present anyway